src/components/ChatForm/ChatForm.tsx

```diff
@@ -71,6 +72,26 @@ export const ChatForm: React.FC<ChatFormProps> = ({
   const pauseReasonsWithPause = useAppSelector(getPauseReasonsWithPauseStatus);
   const [helpInfo, setHelpInfo] = React.useState<React.ReactNode | null>(null);
   const onClearError = useCallback(() => dispatch(clearError()), [dispatch]);
+
+  const { processAndInsertImages } = useAttachedImages();
+  const handlePastingFile = useCallback(
```

Comment:
We might need to have a restriction for attachments within one request. Let's make it 5 for the moment